From 6419df420e7693ecaddbdfb9992091a3d592e19a Mon Sep 17 00:00:00 2001
From: mj <mj@maxlinear.com>
Date: Mon, 24 Nov 2025 07:51:34 -0800
Subject: [PATCH] WLANRTSYS-88756 Reduce BSS Color Threshold Issue: BSS color
 is not changing. Root Cause: After the kernel upgrade, the driver sends 98
 color collision notifications to hostapd within 50 seconds. However, hostapd
 only changes the color if the number of collision events exceeds 100. Fix: As
 a workaround, the threshold value was reduced from 100 to 90 to resolve the
 issue.

---
 src/ap/ap_config.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/ap/ap_config.h b/src/ap/ap_config.h
index d664a691a..d009396c4 100644
--- a/src/ap/ap_config.h
+++ b/src/ap/ap_config.h
@@ -1220,7 +1220,7 @@ enum unii4_config_type
 #define BSS_COLOR_CHANGE_TIMEOUT_RANDOM_SPAN	40
 #define BSS_COLOR_CHANGE_TIMEOUT_MIN			1
 #define BSS_COLOR_CHANGE_TIMEOUT_MAX			120
-#define NUM_BSS_COLOR_COLL_THRESH_DEFAULT		100
+#define NUM_BSS_COLOR_COLL_THRESH_DEFAULT		90
 #define NUM_BSS_COLOR_COLL_THRESH_MIN			1
 #define NUM_BSS_COLOR_COLL_THRESH_MAX			500
 #define BSS_COLOR_COLL_AGE_THRESH_DEFAULT		10
-- 
2.43.0

